zed.advisories.yaml

```diff
@@ -37,6 +37,14 @@ advisories:
             componentType: rust-crate
             componentLocation: /usr/libexec/zed
             scanner: grype
+      - timestamp: 2024-12-09T11:23:12Z
+        type: pending-upstream-fix
+        data:
+          note: |
+            zed has dependencies which rely on different versions of tungstenite.
+            Remediating this vulnerability would require removing dependencies on the older versions.
+            Given multiple versions are intentionally defined, waiting for upstream to remediate.
+            Ref: https://github.com/zed-industries/zed/blob/v0.164.2/Cargo.lock#L13377-L13450
```

Comment:
Raised a question in Slack — I'm not sure I understand what we're saying here yet